cbmc/proofs/mlkem_shake256_rkprf/mlkem_shake256_rkprf_harness.c

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2024 The mlkem-native project authors
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+/*
```

Comment:
I know we're not consistent, but I've turned to just removing all those boilerplate comments for new Makefiles. Can you remove them here too?

Reply:
Ok, we can do this another time.